tutorials/plugins/android/android_plugin.rst

```diff
@@ -251,6 +254,30 @@ Java will most likely run in a separate thread, so calls are deferred:
 Godot will detect this singleton and initialize it at the proper time.
 
 
+Using it from GDScript
+^^^^^^^^^^^^^^^^^^^^^^
+
+First you will need to add your singleton into the android modules to be loaded.
+
+Go to "Project > Project Settings"
+
+Then on the tab "General" go to the "Android" section, and fill the Modules part with your module name.
+
+The module should include the full Java path. For our example: org/godotengine/godot/MySingleton
+
+.. image:: img/android_modules.png
+
+
+Then, from your script:
+
+.. code:: gdscript
+
+    if Engine.has_singleton("MySingleton"):
+        var singleton = Engine.get_singleton("MySingleton")
+        print(singleton.myFunction("World"))
+
+
+
```

Comment:
You used up your empty lines quota :)

A single empty line is enough to separate paragraphs and sections, we have headings to make separations clear.

Reply:
Updated. Hope it's ok now.